aleth-key/KeyAux.h

```diff
@@ -1,21 +1,8 @@
 #pragma once
 
-/*
-	This file is part of cpp-ethereum.
-
-	cpp-ethereum is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 3 of the License, or
-	(at your option) any later version.
-
-	cpp-ethereum is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
-
-	You should have received a copy of the GNU General Public License
-	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
-*/
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 Aleth Authors.
+// Licensed under the GNU General Public License, Version 3.
 /** @file KeyAux.cpp
```

Comment:
The whole comment block should be removed if there is nothing by only `@file`, `@author` and `@data` tag.

Reply:
/** @file KeyAux.cpp
* @author Gav Wood <i@gavwood.com>
 * @date 2014
These Should be Removed ?

Reply:
Yea that block should be removed...it’s in the wrong file anyway